.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,43 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
```

Comment:
maybe you want to add java, kotlin, objective-c and swift?

Reply:
Yes this will be added soon in the future. I was having build issues while compiling the android and iOS app. For now since around 95% of our code is on Javascript, I have enabled this as the first step. I will submit a separate PR once the issue is resolved. Thanks for asking.